### ISSUE TITLE
Section numbering in result pages is broken

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#1742]
The analysis 14 result page at:

https://www.atlas.aei.uni-hannover.de/~julian.westerweck/LSC/O2/analysis14_run_full3/8._workflow/8.03_version/

has broken section numbering. This may be something peculiar about @micamu's installation in:
```
source /home/miriam.cabero/virtualenvs/pycbc-release/bin/activate
```
but should be checked before 1.7.2 is released.

**Comment by spxiwh** [comment]
I've confirmed this is caused by specific versions of Jinja. It works normally on 2.7 and does not work on 2.9.5. For now I don't know why, but a short-term fix would be to *require* say version 2.7.3 (that's what we have on vulcan).

**Comment by spxiwh** [comment]
I've now identified the problem, it's in the syntax of the Jinja template file for two_column layout. This page gives some background, and indeed the first "answer" is actually what we've implemented, which doesn't work:

https://stackoverflow.com/questions/7537439/how-to-increment-a-variable-on-a-for-loop-in-jinja-template

But if you read the comments for that answer:

```
FYI, setting variables across a loop boundary was never supposed to work and was "fixed" in Jinja 2.9, so it won't work anymore. See the other answer for a workaround and github.com/pallets/jinja/issues/641 for a possibly upcoming feature that would solve this properly.
```

ie. we were using a bug in the two_column.html template, not a feature, to make this work.

The second solution in this article does seem a little hacky, but I've demonstrated that it fixes the problem in a small test case. I'll implement this in the two_column.html file tomorrow and test in Miriam's bank_verificator workflow.

.... For the record, this part of Jinja syntax makes *no* sense!
